models/tasks/activity_test.go

```diff
@@ -0,0 +1,47 @@
+package tasks // import github.com/mlimaloureiro/golog/models/tasks
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func timeFromString(str string) time.Time {
+	date, _ := time.Parse(time.RFC3339, str)
+	return date
+}
+
+func TestTaskActivity(t *testing.T) {
+	t.Run("IsRunning()", func(t *testing.T) {
+		t.Run("running", func(t *testing.T) {
+			taskActivity := TaskActivity{StartDate: timeFromString("2019-01-01T10:00:00Z")}
+			assert.True(t, taskActivity.IsRunning())
+		})
+
+		t.Run("not running", func(t *testing.T) {
+			taskActivity := TaskActivity{StartDate: timeFromString("2019-01-01T10:00:00Z"), EndDate: timeFromString("2019-01-02T10:00:00Z")}
```

Comment:
we should align on max columns p/ line and how/when to break lines. usually we have either 100 or 120 max columns

Reply:
also add this to our go guidelines, just saw we don't have it

Reply:
https://github.com/fsschmitt/kilabs-engineering/pull/3